projects/example-app/src/app/auth/effects/auth.effects.ts

```diff
@@ -15,6 +17,26 @@ import { AuthService } from '../services/auth.service';
 
 @Injectable()
 export class AuthEffects {
+  @Effect()
```

Comment:
This effect isn't needed just to map to another action. Change the `login$` effect to use the `LoginPage.Login` action instead.